website/docs/quick-start.md

```diff
@@ -118,28 +98,34 @@ The update all functionality allows you to address discrepancies that may arise
     ```
    You can stop the container by opening a new shell instance and running: `docker stop nginx-test`
 
-### Patch all outdated packages:
-1. Run Copa with a buildkit connection as described above, and omit the report flag:
+### Patch with an optional scanner report
+
+By default, Copa updates all oudated packages. However, you can chose to have a targeted patching of your image by providing an optional vulnerability report.
+
+This example walks through the steps using a Trivy report to patch only packages found vulnerable:
+
+1. Ensure buildkit connection is set up as shown above.
+
+2. Scan the container image for patchable OS vulnerabilities, outputting the results to a JSON file:
+
+    ```bash
+    trivy image --vuln-type os --ignore-unfixed -f json -o nginx.1.21.6.json docker.io/library/nginx:1.21.6
+    ```
+
+    You can also see the existing patchable vulnerabilities in table form on the shell with:
 
     ```bash
-        copa patch -i docker.io/library/nginx:1.21.6
+    trivy image --vuln-type os --ignore-unfixed docker.io/library/nginx:1.21.6
     ```
-2. Run the container to verify that the image has no regressions:
+
+3. To patch the image, supply the Trivy report as an additional argument to the Copa command.
```

Comment:
```suggestion
3. To patch the image, supply the Trivy report as an additional argument to the Copa command. By default, this will produce a tag with a `-patched` suffix.
```